app-antivirus/fangfrisch/fangfrisch-1.0.1.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{7,8} )
+
+inherit distutils-r1
+
+DESCRIPTION="Update and verify unofficial Clam Anti-Virus signatures"
+HOMEPAGE="https://github.com/rseichter/fangfrisch https://pypi.org/project/fangfrisch/"
+SRC_URI="https://github.com/rseichter/fangfrisch/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-python/requests-2.22.0[${PYTHON_USEDEP}]
+	>=dev-python/sqlalchemy-1.3.11[${PYTHON_USEDEP}]"
+RDEPEND="${DEPEND}"
+
+distutils_enable_tests unittest
+
+function python_prepare_all() {
+	sed -i -E -e 's/(SQLAlchemy >= 1.3.1)3/\11/' setup.py || die
```

Comment:
This sed is horribly unreadable and easy to break. Why not strip the dep entirely? it's not like setup.py deps have any value to our users.

Reply:
I'd rather not mess with setup.py at all, but I am blocked by [this issue](https://bugs.gentoo.org/710478). As soon as it is resolved, I will drop the `sed` call.

Reply:
It's probably really better to just "sed" setup.py to forget about SQLAlchemy and specify the dependency in the ebuild instead.

Reply:
Sed now removes the whole line from setup.py.